Src/AutoFixture.NUnit3.UnitTest/ThrowingStubFixture.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    /// <summary>
+    /// A <see cref="IFixture"/> for the benefit of creating stubs of <see cref="AutoDataAttribute"/> for unit testing
+    /// </summary>
+    public class ThrowingStubFixture : IFixture
```

Comment:
Most other hand-written [Test Doubles](http://xunitpatterns.com/Test%20Double.html) in the AutoFixture codebase are named like *DelegatingXyz*, so it might help decreasing the cognitive load if we just name this as `DelegatingFixture` or similar.

What do you think? :smiley: